css-two-column-layout/styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  font-family: Karla, sans-serif;
+
+  /* font-style: normal;
+  src: url("https://fonts.gstatic.com/s/karla/v7/qkBbXvYC6trAT7RVLtyU5rZP.woff2") format("woff2"); */
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.header-content {
+  width: 100%;
+  text-align: center;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.content-section-div {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.content-section-one-fourth {
+  flex-basis: calc(100% / 2);
+  display: flex;
```

Comment:
If you find yourself adding `display: flex` to a `column` class element, that almost always means that the `column` element should have a child `row` class element, with `column` class elements inside of it.

Reply:
Hi, just a heads up, I re-submitted, but didnt see the other change comment. Will go back in to make corrections today.